examples/nlp/lasertagger/official_lasertagger/bert_example.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file contains code artifacts adapted from the original implementation:
+https://github.com/google-research/lasertagger/blob/master/bert_example.py
+"""
+
+"""Build BERT Examples from text (source, target) pairs."""
+
+import collections
+
+from examples.nlp.lasertagger.official_lasertagger import tagging
+
+from nemo.collections.nlp.data.tokenizers import bert_tokenizer
+
+
+class BertExample(object):
+    """Class for training and inference examples for BERT.
+
+  Attributes:
+    editing_task: The EditingTask from which this example was created. Needed
+      when realizing labels predicted for this example.
+    features: Feature dictionary.
+  """
```

Comment:
there is something weird with the indentation...

did black formatted it in this way?

Reply:
yeah, I ran them all through style --fix

Reply:
Okay, all indentations resolved.

Reply:
perfect! 👍 

btw. are you using black formatter? Has it allow you to have those indents...?